src/Form/inputStyles.js

```diff
@@ -49,7 +57,7 @@ const getIconStylesBase = (input, form) => {
 const getIconStylesAutofilled = (input, form) => {
     const icon = getIcon(input, form, 'filled')
 
-    const iconStyle = icon ? {'background-image': `url(${icon}`} : {}
+    const iconStyle = icon ? getBasicStyles(input, icon) : {}
```

Comment:
This was needed because a site was clearing the previous style, thus messing up with our icon (background was repeating infinitely). This re-applies all the styles.